assets/js/components/Event.jsx

```diff
@@ -1,10 +1,26 @@
 import React from 'react'
 import PropTypes from 'prop-types'
+import Card from './spectre/Card'
 
-const Event = ({title}) => <div>{title}</div>
+const Event = ({title, subtitle, description, thumbnail}) => <Card>
```

Comment:
slight preference for enclosing in parens and putting `<Card>` on its own line

Reply:
:+1: